packages/lib/src/components/FeedbackWidget/FeedbackWidget.tsx

```diff
@@ -182,14 +209,28 @@ export function FeedbackWidget({
   function onSubmit(form: FeedbackInput): void {
     if (!currentUser) return
 
+    const formattedAdditionalInformation: AdditionalInformation = {}
+
+    if (additionalInformation) {
+      Object.keys(additionalInformation).forEach(key => {
+        const value = additionalInformation[key]
+        if (typeof value === 'string') {
+          formattedAdditionalInformation[key] = value
+        } else if (Array.isArray(value)) {
+          formattedAdditionalInformation[key] = value.join(', ')
+        }
+      })
+    }
+
```

Comment:
What is the difference between this logic and the logic at line 133 and onwards? It seems that there are some repetitions.

Reply:
Yes, true. I think we don't need to set the values for the additional properties, since it is optional. So I removed the the logic in line 133 and the following.